x-pack/plugins/ml/public/application/settings/anomaly_detection_settings.tsx

```diff
@@ -0,0 +1,229 @@
+                </small>
+              </p>
+            </EuiTextColor>
+            <EuiSpacer size="m" />
+            <EuiFlexGroup alignItems="center">
+              {canGetCalendars && (
+                <EuiFlexItem grow={false} style={{ display: 'block' }}>
+                  <EuiText>
+                    <FormattedMessage
+                      id="xpack.ml.settings.anomalyDetection.calendarsSummaryCount"
+                      defaultMessage="You have {calendarsCountBadge} {calendarsCount, plural, one {calendar} other {calendars}}"
+                      values={{
+                        calendarsCountBadge: <EuiBadge>{calendarsCount}</EuiBadge>,
+                        calendarsCount,
+                      }}
+                    />
+                  </EuiText>
+                </EuiFlexItem>
+              )}
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmpty
+                  data-test-subj="mlCalendarsMngButton"
+                  flush="left"
+                  size="l"
+                  color="primary"
+                  href="#/settings/calendars_list"
+                  isDisabled={canGetCalendars === false}
+                >
+                  <FormattedMessage
+                    id="xpack.ml.settings.anomalyDetection.manageCalendarsLink"
+                    defaultMessage="Manage"
+                  />
+                </EuiButtonEmpty>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmpty
+                  data-test-subj="mlCalendarsCreateButton"
+                  flush="left"
+                  size="l"
+                  color="primary"
+                  href="#/settings/calendars_list/new_calendar"
+                  isDisabled={canCreateCalendar === false}
+                >
+                  <FormattedMessage
+                    id="xpack.ml.settings.anomalyDetection.createCalendarLink"
+                    defaultMessage="Create"
+                  />
+                </EuiButtonEmpty>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem grow={5}>
+            <EuiTitle size="s">
+              <h3>
+                <FormattedMessage
+                  id="xpack.ml.settings.anomalyDetection.filterListsTitle"
+                  defaultMessage="Filter Lists"
+                />
+              </h3>
+            </EuiTitle>
+            <EuiSpacer size="s" />
+            <EuiTextColor color="subdued">
+              <p>
+                <small>
+                  Filter lists contain values that you can use to include or exclude events from the
+                  machine learning analysis.
+                </small>
+              </p>
+            </EuiTextColor>
+            <EuiSpacer size="m" />
+            <EuiFlexGroup alignItems="center">
+              {canGetFilters && (
+                <EuiFlexItem grow={false} style={{ display: 'block' }}>
```

Comment:
same here, you probably should conditionally change `grow` prop instead of overriding `display` style.

Reply:
The `display:block` wasn't needed. I removed it in https://github.com/elastic/kibana/pull/68663/commits/e67c1fb40fd8180d877148fc18d379610b21e5d6